deno_webgpu/compute_pass.rs

```diff
@@ -91,14 +91,15 @@ pub fn op_webgpu_compute_pass_dispatch_workgroups_indirect(
 #[serde]
 pub fn op_webgpu_compute_pass_end(
     state: &mut OpState,
+    #[smi] _command_encoder_rid: ResourceId,
```

Comment:
this parameter is unused in this function, but it is part of the denoland implementation of deno_webgpu, and it's expected to be present by `01_webgpu.js` in this repository

https://github.com/gfx-rs/wgpu/blob/61b7063d8e026415cd840d7097882c16304dc90b/deno_webgpu/01_webgpu.js#L4383-L4386